portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -49,23 +49,28 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
I think you can use `setLimit(maxComments)` to add a limit to your query, then you wouldn't need to increment and check `maxComments <= i` below.

Reply:
As Shandy pointed to me, I was looking at the wrong documentation for Query, looks like there's no support for limit for the Query you guys are using, so let's keep the counter approach for now.